tests/test_morphsqueeze.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+from numpy.polynomial import Polynomial
+from scipy.interpolate import interp1d
+
+
+@pytest.mark.parametrize(
+    "squeeze_coeffs",
+    [
+        # The order of coefficients is [a0, a1, a2, ..., an]
+        # Negative cubic squeeze coefficients
+        [-0.2, -0.01, -0.001, -0.001],
+        # Positive cubic squeeze coefficients
+        [0.2, 0.01, 0.001, 0.001],
+        # Positive and negative cubic squeeze coefficients
+        [0.2, -0.01, 0.001, -0.001],
+        # Quadratic squeeze coefficients
+        [-0.2, 0.005, -0.003],
+        # Linear squeeze coefficients
+        [0.1, 0.3],
+        # 4th order squeeze coefficients
+        [0.2, -0.01, 0.001, -0.001, 0.0001],
+    ],
+)
+def test_morphsqueeze(squeeze_coeffs):
+    # Uniform x-axis grid. This is the same x-axis for all data.
```

Comment:
The comments in the test are not needed, just the comments summarizing the behavior above are useful.